docs/FAQ.md

```diff
@@ -23,3 +23,17 @@ Ignition JSON directly.
 Today, the MCO only blocks on upgrades of control plane nodes.  `oc get clusterversion` effectively reports the version of the control plane. 
 
 To watch rollout of worker nodes, you should look at `oc describe machineconfigpool/worker` (as well as other custom pools, if any).
+
+## Q: How does this relate to Machine API?
+
+Confusingly, there are two fundamental operators in OpenShift 4 that both include "machine" in their name:
```

Comment:
Strike this para.

Reply:
or at least "confusingly"

Reply:
Or maybe "It may seem confusing but there are two..."